nautobot/core/templates/admin/base.html

```diff
@@ -8,6 +8,9 @@
 <html lang="en">
 <head>
     <title>{% block title %}Admin Home{% endblock %} - Nautobot</title>
+    <link rel="stylesheet"
+          href="{% static 'css/hide_body.css' %}"
```

Comment:
Possibly?

```suggestion
          href="{% static 'css/hide_body.css' %}?v{{ settings.VERSION }}"
```

Same comment elsewhere this file is referenced. IIRC this is a mechanism to ensure that when upgrading to a new version of Nautobot the browser is hinted to re-download this file rather than relying on cache, in case it changed in the upgrade.